actionpack/lib/action_dispatch/http/content_security_policy.rb

```diff
@@ -175,7 +175,7 @@ def plugin_types(*types)
     end
 
     def report_uri(uri)
-      @directives["report-uri"] = [uri]
+      @directives["report-uri"] = [uri] if uri
```

Comment:
If have a setting like the original issue, this `uri` will be` Proc` object. Therefore, since here `if` is always true, `report_uri` is always set.
I think that it is necessary to check whether the value is nil after evaluating Proc.